DTS-STN/c19-benefits-node
# C19BF-99 Update Situation Question

# Description
Updated question with request from client.
![image](https://user-images.githubusercontent.com/52539578/156202220-7b8fbc8d-06ed-47f8-bc5a-c3c4e47d1c38.png)

## Test Instructions
1. Download branch and do npm i
2. Run the option listed above should not be on the list of options
3. Updated package.json dependencies with the non-breaking updates. there are still some vulnerabilities but those are breaking changes.  

## Checklist
